examples/basic/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+deno run -c ./tsconfig.json -A --unstable ./mod.ts
```

Comment:
Deno is cross-platform so I'd rather avoid including shell scripts that may seem required to run a Deno DOM-dependent script